File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/PatientTriageMap.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace gov.va.medora.mdo.domain.sm
{
    [Serializable]
    public class PatientTriageMap : BaseModel
    {
        private TriageRelation _triageRelation;

        public TriageRelation TriageRelation
        {
            get { return _triageRelation; }
            set { _triageRelation = value; }
        }
        private Patient _patient;

        public Patient Patient
        {
            get { return _patient; }
            set { _patient = value; }
        }
    }
}

File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/trash/PatientTriageMap.java
package gov.va.med.mhv.sm.model;

import java.io.Serializable;

public class PatientTriageMap extends BaseModel implements Serializable {

	
	/**
	 * 
	 */
	private static final long serialVersionUID = -4585150110378022297L;
	private TriageRelation triageRelation;
	private Patient patient;
	
	
	public TriageRelation getTriageRelation() {
		return triageRelation;
	}
	public void setTriageRelation(TriageRelation triageRelation) {
		this.triageRelation = triageRelation;
	}
	public Patient getPatient() {
		return patient;
	}
	public void setPatient(Patient patient) {
		this.patient = patient;
	}
	
	
}
